app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb

```diff
@@ -63,6 +56,14 @@ def destroy
       respond :resource => @puppet_module
     end
 
+    def index_relation
+      puppet_modules = ContentViewPuppetModule.where(:content_view_id => @view)
+      puppet_modules = puppet_modules.where(:name => params[:name]) if params[:name]
+      puppet_modules = puppet_modules.where(:uuid => params[:uuid]) if params[:uuid]
+      puppet_modules = puppet_modules.where(:author => params[:author]) if params[:author]
```

Comment:
I see you have an author param in the index, but I don't see it declared in the model on scoped search. Did you want to do scoped search on author?